src/main/java/io/vertx/core/http/impl/HttpClientImpl.java

```diff
@@ -120,6 +112,7 @@
 
 
   private final VertxInternal vertx;
+  private final ContextInternal context;
```

Comment:
the client should not depend on a specific context

Reply:
what's the issue? it would depend on a context only when shared